src/Extension.Tests/OutputEvaluationTests.cs

```diff
@@ -12,21 +12,13 @@
 
 namespace Extension.Tests
 {
-    public class OutputEvaluationTests
+    public class OutputEvaluationTests : ProgressiveLearningTestBase
     {
-        private Challenge GetEmptyChallenge()
-        {
-            return new Challenge(new EditableCode[] { });
-        }
-
         [Fact]
         public async Task teacher_can_provide_challenge_evaluation_feedback()
         {
             var lesson = new Lesson();
-            using var kernel = new CompositeKernel
-            {
-                new CSharpKernel()
-            }.UseLessonEvaluateMiddleware(lesson);
+            using var kernel = CreateKernel(lesson);
             var challenge = GetEmptyChallenge();
             await lesson.StartChallengeAsync(challenge);
```

Comment:
same here, starting a challenge that is not part of a lesson should not be allowed